tests/remote_point_evaluation/remote_point_evaluation_01.cc

```diff
@@ -0,0 +1,555 @@
+    background_tria.refine_global(background_n_global_refinements);
+
+  FESystem<spacedim>   background_fe(FE_Q<spacedim>{background_fe_degree},
+                                   spacedim);
+  DoFHandler<spacedim> background_dof_handler(background_tria);
+  background_dof_handler.distribute_dofs(background_fe);
+
+  MappingQ1<spacedim> background_mapping;
+
+  VectorType force_vector_sharp_interface(
+    create_partitioner(background_dof_handler));
+
+  // write computed vectors to Paraview
+  if (false)
+    {
+      GridOut().write_mesh_per_processor_as_vtu(tria, "grid_surface");
+      GridOut().write_mesh_per_processor_as_vtu(background_tria,
+                                                "grid_background");
+    }
+
+  compute_force_vector_sharp_interface(mapping,
+                                       dof_handler,
+                                       dof_handler_dim,
+                                       QGauss<dim>(fe_degree + 1),
+                                       background_mapping,
+                                       background_dof_handler,
+                                       1.0,
+                                       normal_vector,
+                                       curvature_vector,
+                                       force_vector_sharp_interface);
+
+  force_vector_sharp_interface.update_ghost_values();
+
+  // write computed vectors to Paraview
+  if (true)
+    {
+      DataOutBase::VtkFlags flags;
+      // flags.write_higher_order_cells = true;
+
+      DataOut<dim, DoFHandler<dim, spacedim>> data_out;
+      data_out.set_flags(flags);
+      data_out.add_data_vector(dof_handler, curvature_vector, "curvature");
+      data_out.add_data_vector(dof_handler_dim, normal_vector, "normal");
+
+      data_out.build_patches(mapping,
+                             fe_degree + 1,
+                             DataOut<dim, DoFHandler<dim, spacedim>>::
+                               CurvedCellRegion::curved_inner_cells);
+      data_out.write_vtu_with_pvtu_record("./",
+                                          "data_surface",
+                                          0,
+                                          MPI_COMM_WORLD);
+    }
+
+  if (true)
+    {
+      DataOutBase::VtkFlags flags;
+      flags.write_higher_order_cells = true;
+
+      DataOut<spacedim> data_out;
+      data_out.set_flags(flags);
+      data_out.attach_dof_handler(background_dof_handler);
+      data_out.add_data_vector(background_dof_handler,
+                               force_vector_sharp_interface,
+                               "force");
+
+      data_out.build_patches(background_mapping, background_fe_degree + 1);
+      data_out.write_vtu_with_pvtu_record("./",
+                                          "data_background",
+                                          0,
+                                          MPI_COMM_WORLD);
+    }
+
+  force_vector_sharp_interface.print(deallog.get_file_stream());
```

Comment:
How do you check for correctness?

Reply:
I took a look at the result in Paraview. Furthermore, I use the vector as the right-hand-vector in a Navier-Stokes solver and the resulting pressure jump (the derived quantity) is correct there.